k8s/claim/async_deprovision.go

```diff
@@ -28,6 +28,18 @@ func pollDeprovisionState(
 	pollNum := 0
 	pollState := mode.LastOperationStateInProgress
 	pollErrCount := 0
+
+	cfg, err := getConfig()
```

Comment:
I looked... we seem to be doing that in the `main` package and in each of the three modes.

I want this to be mode independent, which is why it's not with any mode configs.

I _could_ put it with the root config in `main`. There are probably pros and cons to that:

Pro: It's mode-agnostic, so keep it with the most general config object we have.

Con: I generally like to keep config closer to where it's used / what it affects.

Thoughts?